app/view/components/StyledText.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {I18nManager} from 'react-native';
+
+import {useTheme, Text} from 'react-native-paper';
+
+const StyledText = (props) => {
+  const {colors, fonts} = useTheme();
+  const {textColor = colors.text, family = 'regular', style, ...rest} = props;
+
+  const font = fonts[family];
+  const writingDirection = I18nManager.isRTL ? 'rtl' : 'ltr';
+
+  return (
+    <Text
+      {...rest}
+      style={[
+        {color: textColor, ...font, textAlign: 'left', writingDirection},
+        style,
+        props.style,
```

Comment:
What is the difference between lines 18 and 19? both are the same. I think so.

Reply:
You are right, it has been duplicated